table/block_based/filter_block_reader_common.cc

```diff
@@ -28,12 +28,12 @@ Status FilterBlockReaderCommon<TBlocklike>::ReadFilterBlock(
   const BlockBasedTable::Rep* const rep = table->get_rep();
   assert(rep);
 
-  const Status s =
-      table->RetrieveBlock(prefetch_buffer, read_options, rep->filter_handle,
-                           UncompressionDict::GetEmptyDict(), filter_block,
-                           get_context, lookup_context,
-                           /* for_compaction */ false, use_cache,
-                           /* async_read */ false);
+  const Status s = table->RetrieveBlock(
+      prefetch_buffer, read_options, rep->filter_handle,
+      UncompressionDict::GetEmptyDict(), filter_block, get_context,
+      lookup_context,
+      /* for_compaction */ false, use_cache,
+      /* async_read */ false, /* use_block_cache_for_lookup */ true);
```

Comment:
nit: While I feel this is totally fine, I believe that we introduce negative option as default in general. Name the boolean as `skip_block_cache_lookup` and look up when it's `false`